vite.config.js

```diff
@@ -0,0 +1,11 @@
+import { defineConfig } from 'vite';
+import reactRefresh from '@vitejs/plugin-react-refresh';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+    build: { target: 'esnext' },
+    optimizeDeps: {
+        include: ['gosling.js']
```

Comment:
why is this necessary?

Reply:
It is not necessary. It was left over after testing something.